golonomic.go

```diff
@@ -18,36 +18,52 @@ const (
 
 var inverse = mat.NewDense(3, 3, nil)
 
+var motorA *ev3dev.TachoMotor
+var motorB *ev3dev.TachoMotor
+var motorC *ev3dev.TachoMotor
+
 func setupInverse() {
 	data := []float64{
-		math.Cos(a1*math.Pi/180), math.Cos(a2*math.Pi/180), math.Cos(a3*math.Pi/180),
-		math.Sin(a1*math.Pi/180), math.Sin(a2*math.Pi/180), math.Sin(a3*math.Pi/180),
+		math.Cos(a1 * math.Pi / 180), math.Cos(a2 * math.Pi / 180), math.Cos(a3 * math.Pi / 180),
+		math.Sin(a1 * math.Pi / 180), math.Sin(a2 * math.Pi / 180), math.Sin(a3 * math.Pi / 180),
 		1, 1, 1}
 	matrix := mat.NewDense(3, 3, data)
 
 	err := inverse.Inverse(matrix)
 	if err != nil {
 		log.Fatalf("failed to inverse matrix: %v", err)
 	}
-
-	fn := mat.Formatted(inverse, mat.Prefix("    "), mat.Squeeze())
-	fmt.Printf("i = %v", fn)
 }
 
-func testMotors() {
-	outA, err := ev3dev.TachoMotorFor("ev3-ports:outA", "lego-ev3-l-motor")
+func setupMotors() {
+	var err error
+
+	motorA, err = ev3dev.TachoMotorFor("ev3-ports:outA", "lego-ev3-l-motor")
```

Comment:
You could probably create a helper function e.g. ```motorA := initMotor("A")```. In this case error is not necessary if you're just going to exit the program within initMotor